### PR TITLE
Create sort.py

### PR DESCRIPTION
**Pull request by inyong37**
- bogo sort
- slow sort
- is_sorted 내부 함수

### CODE REVIEW

**Comment by inyong37** [comment]
TODO

Evil Sort

https://web.archive.org/web/20131210012102/http://richardhartersworld.com/cri_d/cri/2001/badsort.html#evilsort

Sleep Sort

https://www.geeksforgeeks.org/sleep-sort-king-laziness-sorting-sleeping/

File: sort/sort.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  '''
+  return all(a <= b for a, b in zip(l, l[1:]))
+
+
+def bogo_sort(l: List[int]) -> List[int]:
+  '''
+  bogo sort:  https://en.wikipedia.org/wiki/Bogosort
+  '''
+  while not __is_sorted__(l):
+    shuffle(l)
```
Comment (SNURFER) [inline sort/sort.py:17]: 오우 O(n!)

**Comment by Avantgarde95** [comment]
LGTM

**Comment by inyong37** [comment]
자체 머지!

**Pull request merged**